### ISSUE TITLE
Error when attempting to reset

### ISSUE THREAD

**Issue by 0x8890** [substack/psy#14]
I tried running `psy reset` and keep getting this error:

```
TypeError: path must be a string
    at TypeError (native)
    at Object.fs.unlink (fs.js:975:11)
    at /usr/local/lib/node_modules/psy/index.js:103:8
    at /usr/local/lib/node_modules/psy/node_modules/auto-daemon/index.js:24:18
    at /usr/local/lib/node_modules/psy/node_modules/auto-daemon/index.js:77:9
    at f (/usr/local/lib/node_modules/psy/node_modules/once/once.js:17:25)
    at Socket.<anonymous> (/usr/local/lib/node_modules/psy/node_modules/auto-daemon/index.js:84:9)
    at Socket.g (events.js:273:16)
    at emitNone (events.js:85:20)
    at Socket.emit (events.js:179:7)
```

There are definitely processes running, when I run `psy ls` it works as expected. 

This error happened on FreeBSD 10.2 using Node.js 5.9.1 if that helps.

**Comment by karissa** [comment]
thanks found that bug and added a test

**Comment by 0x8890** [comment]
@karissa that was super fast, thanks!

**Comment by karissa** [comment]
no problem! fixed in 1.7.2
